bindinfo/cache.go

```diff
@@ -191,3 +207,79 @@ func (b cache) appendNode(newBindRecord *bindRecord, sparser *parser.Parser) err
 	b[hash] = append(b[hash], newNode)
 	return nil
 }
+
+// AddGlobalBind implements GlobalBindAccessor.AddGlobalBind interface.
+func (h *Handle) AddGlobalBind(originSQL, bindSQL, defaultDB, charset, collation string) error {
```

Comment:
Add a `ctx context.Context` argument here?

Reply:
ctx has been created in this method

Reply:
We just need a ctx without any reason

Reply:
If so, we do not need to create a new `ctx`, just pass a `ctx` from the outside.